src/views/Governance/components/ProposalPage/components/PollDetailsTab.tsx

```diff
@@ -1,15 +1,26 @@
-import { Grid, Typography } from "@mui/material";
+import { Grid } from "@mui/material";
 import { TextButton } from "@olympusdao/component-library";
+import MarkdownIt from "markdown-it";
 import { ProposalTabProps } from "src/views/Governance/interfaces";
 
 export const PollDetailsTab = ({ proposal }: ProposalTabProps) => {
+  const getRenderedDetails = () => {
+    console.log(proposal?.content);
+    return {
+      __html: MarkdownIt({ linkify: true, typographer: true }).render(proposal ? proposal.content : ""),
+    };
+  };
+
   return (
     <Grid container direction="column">
       <Grid item>
-        <Typography variant="body1">{proposal.content}</Typography>
+        {/* TODO(appleseed): ... verify this is ok, XSS security wise */}
+        <div dangerouslySetInnerHTML={getRenderedDetails()} />
```

Comment:
we may want to look at https://github.com/remarkjs/react-markdown, as it doesn't require dangerouslySetInnerHTML